### DISCUSSION TITLE
What are supported versions of Redmine?

### DISCUSSION THREAD

**Discussion by ivortep** [sms77io/redmine#1] [General]
Hello,
i`m not able to install plugin on redmine 5.0.4. Could you please describe supported versions. Also is the plugin free or not?
Thanks a lot in advance!

**Comment by matthiez** [comment]
Hi @ivortep,
thanks for reporting.
We are investigating this right now and will get back to you as soon as possible.

**Comment by matthiez** [comment]
Hi @ivortep,
please try with [v0.2.0](https://github.com/seven-io/redmine/releases/download/v0.2.0/seven-redmine-latest.zip).
I just tested it with redmine v5.0.6, it should work with your version too.
The installation instruction has also changed a little.

Hope this helps!

**Reply by ivortep** [reply]
Great, I have successfully installed plugin. Thanks for your quick response. I will read more about the plugin and how can be useful in the system, 
I don`t know that here is the right place for my question, is there possibility the plugin to be integrated with issues, so for any created immediate issue specific users to receive SMS or based on group or something. It will be great. There is no such a future for redmine as far as I familiar

**Reply by matthiez** [reply]
Hi @ivortep,
we are working on your request right now. I will get back to you soon.

**Reply by matthiez** [reply]
Hi @ivortep,
we have added the functionality to send SMS on new issues.
You can choose on which priorities to send.
You can set assignee as the recipient, which will mail the assigned user, or define your own recipient phone number.

Feel free to request any further requirements.

**Reply by ivortep** [reply]
Wow, really quick! Thanks a lot!
Could you please check when is possible for you? rails aborted after upgrade- rails aborted!
LoadError: cannot load such file -- listen.
I`m not able to install/migrate plugin.

**Reply by matthiez** [reply]
Hi @ivortep,
please try again with the latest [release](https://github.com/seven-io/redmine/releases/download/v0.3.0/seven-redmine-latest.zip).

Hope this helps!

**Reply by ivortep** [reply]
Thanks but still the same:

rails aborted!
LoadError: cannot load such file -- listen
config/environments/development.rb:58:in `block in <top (required)>'
config/environments/development.rb:5:in `<top (required)>'
config/environment.rb:16:in `<top (required)>'
lib/tasks/redmine.rake:73:in `block (2 levels) in <top (required)>'
Tasks: TOP => redmine:plugins:migrate => environment

**Reply by matthiez** [reply]
Hi @ivortep,
please see the updated installation steps in the README.
It should work now.

Hope this helps!

**Reply by ivortep** [reply]
Everything works fine now. Thanks a lot. You are awesome, guys! Wonderful, fast support and service!
